### ISSUE TITLE
TypeError: Cannot read property 'multiple' of undefined

### ISSUE THREAD

**Issue by freakysmurf** [Zazama/node-id3#53]
```
 if(SFrames[SRawToNameMap[tag]].multiple && currentTags[tag] && rawTags[tag]) 
                                           ^

TypeError: Cannot read property 'multiple' of undefined
    at C:\test\nodejs-test\node_modules\node-id3\index.js:284:44
    at Array.map (<anonymous>)
    at NodeID3.update (C:\test\nodejs-test\node_modules\node-id3\index.js:283:30)
    at Object.<anonymous> (C:\test\nodejs-test\test_node-id3.js:12:23)
```
test_node-id3.js:
```
var nodeID3 = require('node-id3');
    
var tags = {
    title: 'title',
    album: 'album'
};

var path = "test.mp3";
console.log( "\nWriting tags: %s to file %s", JSON.stringify(tags), path );

let success = nodeID3.update(tags, path);

console.log(success ? "Ok" : "Failed");
```

**Comment by Zazama** [comment]
Sorry for the inconvenience, fixed with https://github.com/Zazama/node-id3/commit/59ef3c00e35077f358d256e33ec9ca5e12833942 & published to npm

**Comment by freakysmurf** [comment]
Thanx for your fast reply. The same code change has to be done in the else branch (line 312).

**Comment by Zazama** [comment]
Hi, I've answered in the commit: https://github.com/Zazama/node-id3/commit/59ef3c00e35077f358d256e33ec9ca5e12833942

> I really should make async use the sync code :')
> fixed a0353f4

It's published to npm, I'll create a github release now
